GovUkDesignSystem/HtmlGenerators/RadiosHtmlGenerator.cs

```diff
@@ -50,14 +51,18 @@ internal static class RadiosHtmlGenerator
                     HintViewModel itemHint = null;
 
                     radioHints?.TryGetValue(enumValue, out itemHint);
+                    Dictionary<string, string> attributes = null;
```

Comment:
The blank lines here are odd, it looks like you're saying that the two lines above go together, and are separate from the new line below.
Please delete the blank lines between the two hint lines (51 and 53) and the two attribute lines (54 and 56) and add a blank line between 53 and 54.